src/client.rs

```diff
@@ -121,6 +121,7 @@ impl Client {
     /// # movies.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     /// # });
     /// ```
+    #[must_use]
```

Comment:
Yeah, it doesn't look very « rust-idiomatic »

Reply:
They are clippy pedantic suggestions, and I also think it's valid to have must_use in case there is a return type of `Self`:
https://rust-lang.github.io/rust-clippy/master/index.html#return_self_not_must_use

Reply:
reverted